modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -197,6 +197,16 @@ inline IE::Blob::Ptr wrapIE(const cv::MediaFrame::View& view,
 
 template<class MatType>
 inline void copyFromIE(const IE::Blob::Ptr &blob, MatType &mat) {
+    const auto& desc = blob->getTensorDesc();
+    const auto ie_type = toCV(desc.getPrecision());
+    if (ie_type != mat.type()) {
+        std::stringstream ss;
+        ss << "Failed while copying blob from IE to OCV: "
```

Comment:
Failed to copy

Reply:
Also, I doubt if multi-line exception text is useful. At least a single-liner is easier to grep

Reply:
Done